File: OnlineJudge/CodeForces/A/236.cpp
#include <iostream>
#include <cstring>

using namespace std;

int main() {
    string username; cin >> username;

    bool distinctCharacters[26] = {false};
    // Initialize the marks of characters seen

    for (char character : username) {
        // Assign every character in the string to the variable 'character'
        distinctCharacters[character - 'a'] = true;
    }

    int count = 0;
    for (int i = 0; i < 26; i++) {
        if (distinctCharacters[i]) {
            count++;
        }
    }

    if (count % 2) {
        cout << "IGNORE HIM!" << endl;
    } else {
        cout << "CHAT WITH HER!" << endl;
    }
    return 0;
}

File: OnlineJudge/CodeForces/A/236.py
# Problem name: Boy or Girl
# Problem tags: brute force, implementation, strings
# Solve time:   2024.1.14
# Author:       Nulla

username = input().strip()

# Count the number of distinct charachters
distinct_characters = set(username)

# Determine the gender based on the count
if len(distinct_characters) % 2 == 0:
    print("CHAT WITH HER!")
else:
    print("IGNORE HIM!")